### ISSUE TITLE
<Help content> Check WCAG 2.2 minimum target size

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1677]
please refer to #1674

WCAG 2.2 requires a minimum size for a tabbable target, see details in [Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum) and [Understanding Target Size (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
Also, refer to the [team discussion notes](https://ibm.ent.box.com/notes/1304210878893).

**Comment by philljenkins** [comment]
I created a box note for the new rule and started Help content
[WCAG 2.2 rule: element_target_size](https://ibm.ent.box.com/notes/1326676461508)

**Comment by marcjohlic** [comment]
@philljenkins do you have an update on this one?  What are the next steps?

**Comment by philljenkins** [comment]
Working back on the Help. The draft being updated in box for a SME review

[WCAG 2.2 rule: element_target_size](https://ibm.ent.box.com/notes/1326676461508)

**Comment by philljenkins** [comment]
Help ready for build and review
https://github.com/IBMa/equal-access/blob/dev-1674/accessibility-checker-engine/help-v4/en-US/target_spacing_sufficient.html

**Comment by philljenkins** [comment]
@mbgower  When you get a minute, could you peer review the Help for Target size minimum? 
https://ibm.box.com/s/xt8svpudm71vofmtc2hmqu30fahfka04

**Comment by philljenkins** [comment]
Updated fail messages and removed AAA requirement from Help

**Comment by philljenkins** [comment]
Local testing reveals the correct Help pages:
![image](https://github.com/IBMa/equal-access/assets/16193609/c91667e3-0ac6-433a-85c0-0f2d895b18ae)
![image](https://github.com/IBMa/equal-access/assets/16193609/68490d19-bff6-43f5-9b4e-abc924bec72b)

**Comment by ErickRenteria** [comment]
PR has a few conflicts
